docs/stdlib/json.rst

```diff
@@ -75,10 +76,11 @@ Anything in EdgeDB can be cast into :eql:type:`json`:
     db> select <json>cal::to_local_date(datetime_current(), 'UTC');
     {'"2019-04-02"'}
 
-Any :eql:type:`Object` can be cast into :eql:type:`json`. This
-produces the same JSON value as the JSON serialization of that object.
-That is, the result is the same as the output of :eql:stmt:`select
-expression <select>` in *JSON mode*, including the type shape.
+Additionally, any :eql:type:`Object` in EdgeDB can be casted as a
```

Comment:
"cast" is the correct past tense of "cast"

Reply:
Good catch, this may be a reoccurence. Thanks for pointing it out!